_episodes/01-introduction.md

```diff
@@ -83,29 +85,27 @@ called the hardware. One of these pieces is the CPU or processor; another is
 the amount of memory or RAM that your computer can use to store information 
 temporarily while running programs; another is the hard drive, which can store 
 information over the long-term. All these pieces work together to do the 
-"computing" of a computer, but we don't see them, because they're hidden away. 
+"computing" of a computer, but we don't see them because they're hidden from view (usually). 
 
 Instead, what we see is our desktop, program windows, different folders, and 
-files. These all 
-live in what's called the file system. Everything on your computer - programs, 
+files. These all live in what's called the file system. Everything on your computer - programs, 
 pictures, documents - lives somewhere in the file system. One way to think of 
-the file system is the layer of stuff that can be activated to use use the CPU, memory and hard 
-drive of your computer. 
+the file system is the layer of stuff that uses the CPU, memory and hard 
+drive of your computer on your behalf.
 
-NOW, imagine you wanted to have a second computer. You don't want to buy a 
-whole new computer because it's too expensive. What if, instead, you could have 
-another filesystem that you could store and access from your main computer, 
-but that is self-contained? 
+NOW, imagine you wanted to have a second computer but you're a cheapskate and don't want to buy a 
+whole new computer because it's too expensive and bringing two laptops to a meeting is a weird flex. 
+What if, instead, you could have another filesystem that you could store and access from your main computer, 
+but was inside your existing computer? 
```

Comment:
Again a suggested edit to this text...feel free to accept or not, or to modify the suggested text.

```suggestion
NOW, imagine you wanted to install some software but you don't want to make a mess
of your working system by trying to install lots of additional libraries and other
dependencies that are required by the new software. You could get a second computer
but you don't want to buy a whole new computer because it's too expensive. What if,
instead, you could have another, self-contained, filesystem with your new software
installed in it? This filesystem would be stored within, but isolated from, your main
computer system but you'd be able to access and work with it on your main computer
without affecting your existing software or configuration. 
```

Reply:
I like the inclusion of the "messing up" bit - it helps to emphasize the cost of complexity and maintenance in the students' minds.